Source/SwiftLintFramework/Extensions/NSFileManager+SwiftLint.swift

```diff
@@ -31,4 +32,18 @@ extension FileManager: LintableFileManager {
             return nil
         } ?? []
     }
+
+    public func isFile(atPath path: String, modifiedSince: Date) -> Bool {
+        // If there is any problem reading the modification date, prefer to lint the file
+        do {
+            let attr = try attributesOfItem(atPath: path)
+            if let fileModificationDate = attr[FileAttributeKey.modificationDate] as? Date {
+                return fileModificationDate > modifiedSince
```

Comment:
That would be another strategy (honestly I don't have a strong opinion in either direction), but for that we would need to store the previous file modification date for every file. Right now this solution just stores the last run date, so it cannot use a !=. That would be completely doable, though.

I found easier to just be conservative about using the cache or not: in any case this is a just a cache, so we always can throw it out and start again. The only issue that I can think of is in an aberrant situation where the clock is constantly changing. Again, I'm ignorant about if that could really happen :/